src/main/java/io/swagger/codegen/v3/generators/DefaultCodegenConfig.java

```diff
@@ -115,6 +115,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultCodegenConfig.class);
 
     public static final String DEFAULT_CONTENT_TYPE = "application/json";
+    private static final String NOT_NULL_JACKSON_ANNOTATION = "notNullJacksonAnnotation" ;
```

Comment:
this is specific to a generator, not to the whole set, DefaultCodegen must be left unchanged, you would code this into the Java generator

Reply:
done